client/locations-list-map.js

```diff
@@ -683,6 +685,18 @@ function getFlatFilteredEntries(data, filters) {
 
 function getEntryEl(entry) {
   if (!entry.domElem) {
+    // feature flag for email contact form
+    const searchParams = new URLSearchParams(url.search);
+    const showContact = searchParams.get('show-contact') == 'true';
+
+    // feature flag to insert fake contact info for testing
+    const fakeContact = searchParams.get('fake-contact') == 'true';
```

Comment:
do you think it'd be helpful to pass in fakeContact as an encrypted email string for testing? We could send around a few encrypted emails to testers so they could see the full flow rather than everything getting sent to you.

Reply:
Yeah. I was thinking this might be worth doing. I'll go ahead and add it.